src/SearchBar.tsx

```diff
@@ -88,6 +164,10 @@ export const SearchBar: React.FC<SearchBoxProps> = () => {
           reset={<ClearIcon />}
           translations={translations}
         />
+        <VoiceSearch
+          searchAsYouSpeak
```

Comment:
If this is a boolean prop, can you please set it to true explicitly?

Reply:
yes it is a boolean prop.